pegasus/sites.v3/code.org/public/about/jobs.md.erb

```diff
@@ -26,8 +28,13 @@ Launched in 2013, Code.org&reg; is a nonprofit dedicated to expanding access to
 **[Working at Code.org](#info)**
 
 **We are currently hiring for:**
-<div class="hire-jobs"></div>
-<script id="hire-embed-loader" async defer src="https://hire.withgoogle.com/s/embed/hire-jobs.js?company=codeorg"></script>
+<div id='lever-jobs-container'></div>
+<script type='text/javascript'>
```

Comment:
I must defer to @Hamms as usual on what kind of content we like to see in a `.erb`.

Reply:
I think our general philosophy is that `.erb`s are expected to be pretty manually maintained, so we're fine with just about anything being in them. Another option for this JS would be to extract it into a standalone file and include it, but that seems like overkill for what this is.

LGTM!